spec/support/factory_girl.rb

```diff
@@ -1,10 +1,31 @@
 require 'factory_girl'
 
 FactoryGirl.define do
-  factory :user do
-    sequence(:email) {|n| "user#{n}@example.com" }
-    password 'password'
-    password_confirmation 'password'
+  factory :user do |f|
+    f.email { Faker::Internet.email }
+    f.password { Faker::Internet.password(10, 20) }
   end
+end
 
+FactoryGirl.define do
```

Comment:
You only need `FactoryGirl.define do` once - *all* your factories can go inside that one block you defined at the top:

```
FactoryGirl.define do
  factory :user do
    ...
  end

  factory :spot do 
    ...
  end
end
```